ui/src/app/views/workspace-edit/component.css

```diff
@@ -16,3 +16,11 @@
 .checkbox-group {
   padding: 1rem 0;
 }
+
+.red-background {
```

Comment:
I'd prefer to name this `error` or `validation-error` or something -- I think we'll want to follow the pattern you're setting up in lots of places / have a general error style.